File: app/src/main/java/com/neome/java/api/ent/entAside/msg/MsgPaymentVerify.java
// neome.ai API - do not change
//

package com.neome.java.api.ent.entAside.msg;

import com.neome.java.api.nucleus.base.msg.Msg;

@SuppressWarnings("unused")
public class MsgPaymentVerify extends Msg
{
  public String invoiceId;

  public String paymentId;

  public String signature;
}

File: app/src/main/java/com/neome/api/ent/entAside/msg/MsgPaymentVerify.kt
// neome.ai API - do not change
//
@file:Suppress("unused", "MemberVisibilityCanBePrivate", "CanBeParameter", "PropertyName")


package com.neome.api.ent.entAside.msg

import com.neome.api.nucleus.base.msg.Msg

interface MsgPaymentVerify : Msg
{
  val invoiceId: String
  val paymentId: String
  val signature: String
}